### ISSUE TITLE
Doesn't show up as a client for TSX files even though biome is installed

### ISSUE THREAD

**Issue by vikigenius** [cxa/lsp-biome#4]
I checked my node_modules and I do have biome installed

`node_modules/@biomejs/biome/bin/biome`

I use doom emacs and install biome like this

In my packages.el I have

```
(package! lsp-biome :recipe (:host github :repo "cxa/lsp-biome"))
```

And then in my config I have

```
(use-package! lsp-biome)
```

I see that the package is installed because I am able to see commands like

`lsp-biome-fix-all`

But the server is not being started?

**Comment by vikigenius** [comment]
Found the issue looks like 
https://github.com/cxa/lsp-biome/blob/decff4b22e4326c09c287462d44a2be330dd9003/lsp-biome.el#L94

Means that this fails if there is no config file.

**Comment by vikigenius** [comment]
Also after fixing this issue by having a dummy config file. I am getting a 

Error running timer: (void-variable apheleia-formatters)

**Comment by cxa** [comment]
> Means that this fails if there is no config file
Yes we need a config file to work, no plan to support config-less because it's not recommended way.

> Error running timer: (void-variable apheleia-formatters)
We only access this variable when apheleia-mode is running. Check your apheleia config.

**Comment by the9000** [comment]
The fact that the config is needed, and that when the config is absent, the configuration is considered invalid, without any notice.
Maybe if everything else is fine but no `biome.json` can be found the user should see a message.

**Comment by cxa** [comment]
@vikigenius I updated the way to locate config file and executable, please give it a try.
